apollo-gradle-plugin-incubating/src/main/kotlin/com/apollographql/apollo/gradle/api/CompilerParams.kt

```diff
@@ -34,6 +35,13 @@ interface CompilerParams {
    */
   val customTypeMapping: MapProperty<String, String>
 
+  /**
+   * For custom persisted query Ids.
+   *
+   * null by default
```

Comment:
`null by default` is not really explanatory. 

What about this:

```
If not provided, default hashing algorithm (sha256) will be used
```

Reply:
It would be nice to include implementations for Sha1 and Md5: `Md5IdGenerator`, `Sha1IdGenerator`. Can be done in a later PR though